src/OpenTelemetry.Exporter.Prometheus.AspNetCore/PrometheusExporterMeterProviderBuilderExtensions.cs

```diff
@@ -62,9 +62,11 @@ public static MeterProviderBuilder AddPrometheusExporter(
         });
     }
 
-    private static MetricReader BuildPrometheusExporterMetricReader(PrometheusAspNetCoreOptions options)
+    private static BaseExportingMetricReader BuildPrometheusExporterMetricReader(PrometheusAspNetCoreOptions options)
     {
+#pragma warning disable CA2000
```

Comment:
nit: please include a comment here describing what is being disabled.

Reply:
ad047ab81caef9c4a6164b7ec93aef243b60ef1e